delivery_carrier_b2c/README.rst

```diff
@@ -0,0 +1,37 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+ :alt: License: AGPL-3
+
+Delivery Carrier Business To Customer
+=====================================
+
+This module adds specific fields to partner for delivery purpose some carrier services offers to deliver package to any place in a country (ie French So Colissimo).
+In this case addition fields are required with a more complete view on partner
```

Comment:
This line reads as if the fields are required in the partner view but that is not the case. Can you clarify?